en/docs/Administer/ProductAdministration/ManagingUsersAndRoles/ManagingUserStores/writing-a-custom-user-store-manager.md

```diff
@@ -42,50 +56,51 @@ The following list briefly explains the use of the available methods in the `Abs
+<td><p>If you want to change the authentication logic you can override this method and write your own implementation. The default task of this method is to compare the given password with the stored password. The given credentials are passed to the <code>                preparePassword               </code> method to do the salting or encryption before the comparison takes place.</p></td>
 </tr>
 <tr class="even">
-<td><p><code>               String preparePassword(Object password, String saltValue)              </code></p></td>
+<td><p><code>                String preparePassword(Object password, String saltValue)               </code></p></td>
 <td>This returns the encrypted or plain-text password based on the configurations.</td>
 <td><p>You can override this method if you need to change the way you encrypt the password. If you want to change the algorithm that is used for encryption, you can configure it.</p></td>
 </tr>
 <tr class="odd">
-<td><p><code>               Properties getDefaultUserStoreProperties()              </code></p></td>
+<td><p><code>                Properties getDefaultUserStoreProperties()               </code></p></td>
 <td><div class="content-wrapper">
 <p>The default properties of the user store are returned using this method. These properties are used in user store related operations.</p>
-!!! note
-<p>Be sure to manually add the following property when you implement the class:</p>
-<div class="code panel pdl" style="border-width: 1px;">
-<div class="codeContent panelContent pdl">
-<pre class="java" data-syntaxhighlighter-params="brush: java; gutter: false; theme: Confluence" data-theme="Confluence" style="brush: java; gutter: false; theme: Confluence"><code>setOptionalProperty(&quot;Disabled&quot;, &quot;false&quot;, &quot;Whether user store is disabled&quot;);</code></pre>
-</div>
-</div>
-<p>This property is what controls whether the user store is enabled or disabled.</p>
-
+<div class="admonition note">
+<p class="admonition-title">Note</p>
+    <p>Be sure to manually add the following property when you implement the class:</p>
+    <div class="code panel pdl" style="border-width: 1px;">
+    <div class="codeContent panelContent pdl">
+    <div class="sourceCode" id="cb1" data-syntaxhighlighter-params="brush: java; gutter: false; theme: Confluence" data-theme="Confluence" style="brush: java; gutter: false; theme: Confluence"> <pre class="sourceCode java"><code class="sourceCode java"><a class="sourceLine" id="cb1-1" title="1"><span class="fu">setOptionalProperty</span>(<span class="st">&quot;Disabled&quot;</span>, <span class="st">&quot;false&quot;</span>, <span class="st">&quot;Whether user store is disabled&quot;</span>);</a></code></pre>
```

Comment:
Looks like there is some unnecessary Confluence related formatting as well.
Example <div class="sourceCode" id="cb1" data-syntaxhighlighter-params="brush: java; gutter: false; theme: Confluence" data-theme="Confluence" style="brush: java; gutter: false; theme: Confluence">

Reply:
Fix https://github.com/wso2/docs-apim/pull/591/commits/139ca887e93bac06b6979c02d8a13ba69cb88bd9